metadata/proto/metadata.proto

```diff
@@ -200,6 +200,7 @@ message FeatureVariant {
     ComputationMode mode = 18;
     bool is_embedding = 19;
     int32 dimension = 20;
+    string definition = 21;
```

Comment:
Would change this so its like
```  
   bool is_embedding = 19;
   int32 dimension = 20;
   FeatureParameters additional_parameters = 21;
}
message FeatureParameters {
  oneof featuretype {
    PrecomputedFeatureParameters precomputed = 1;
    OndemandFeatureParameters ondemand = 2;
  }
}

OndemandFeatureParameters {
  string definition = 1;
}
```

Reply:
I see, a param bag to hold new props whenever we want. Changing now.